src/main/jte/create_booking.jte

```diff
@@ -41,7 +43,8 @@
                 <span class="label-text-alt">${context.localize("bookings.end")}</span>
             </label>
             <input id="end" name="end" type="datetime-local" class="input input-bordered"
-                   value="${end.toString()}"
+<%--                   value="${end.format(DateTimeFormatter.ISO_DATE_TIME)}"--%>
+                   value="${end.format(DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm"))}"
```

Comment:
This should maybe be changed, only left it here since i couldn't find other working formats